### ISSUE TITLE
Can't install FOSCKEditor bundle with symfony 4.4

### ISSUE THREAD

**Issue by TBourgault** [FriendsOfSymfony/FOSCKEditorBundle#204]
Hello, I tried to install FOSCKEditor with a new Symfony 4.4 (website-skeleton) but I still get the same error message:

```composer require friendsofsymfony/ckeditor-bundle
Using version ^2.1 for friendsofsymfony/ckeditor-bundle
./composer.json has been updated 
Loading composer repositories with package information 
Updating dependencies (including require-dev) 
Restricting packages listed in "symfony/symfony" to "4.4.*"
Your requirements could not be resolved to an installable set of packages.

Problem 1
- Conclusion: remove doctrine/doctrine-bundle 2.0.0
- Conclusion: don't install doctrine/doctrine-bundle 2.0.0 
- twig/twig v2.0.0 conflicts with doctrine/doctrine-bundle[2.0.0]. 
- twig/twig v2.1.0 conflicts with doctrine/doctrine-bundle[2.0.0]. 
- twig/twig v2.2.0 conflicts with doctrine/doctrine-bundle[2.0.0]. 
- twig/twig v2.3.0 conflicts with doctrine/doctrine-bundle[2.0.0]. 
- twig/twig v2.3.1 conflicts with doctrine/doctrine-bundle[2.0.0]. 
- twig/twig v2.3.2 conflicts with doctrine/doctrine-bundle[2.0.0]. 
- Installation request for doctrine/doctrine-bundle (locked at 2.0.0) -> satisfiable by doctrine/doctrine-bundle[2.0.0]. 
- Conclusion: don't install knplabs/knp-menu 2.6.0|install twig/twig v2.0.0|install twig/twig v2.1.0|install twig/twig v2.2.0|install twig/twig v2.3.0|install twig/twig 
v2.3.1|install twig/twig v2.3.2
- Conclusion: remove knplabs/knp-menu 2.6.0|install twig/twig v2.0.0|install twig/twig v2.1.0|install twig/twig v2.2.0|install twig/twig v2.3.0|install twig/twig v2.3.1|
install twig/twig v2.3.2 
- twig/twig v2.7.1 conflicts with knplabs/knp-menu[2.6.0]. 
- twig/twig v2.7.2 conflicts with knplabs/knp-menu[2.6.0]. 
- twig/twig v2.7.3 conflicts with knplabs/knp-menu[2.6.0]. 
- twig/twig v2.7.4 conflicts with knplabs/knp-menu[2.6.0]. 
- twig/twig v2.8.0 conflicts with knplabs/knp-menu[2.6.0]. 
- twig/twig v2.8.1 conflicts with knplabs/knp-menu[2.6.0]. 
- twig/twig v2.4.0 conflicts with knplabs/knp-menu[2.6.0]. 
- twig/twig v2.4.1 conflicts with knplabs/knp-menu[2.6.0]. 
- twig/twig v2.4.2 conflicts with knplabs/knp-menu[2.6.0]. 
- twig/twig v2.4.3 conflicts with knplabs/knp-menu[2.6.0]. 
- twig/twig v2.4.4 conflicts with knplabs/knp-menu[2.6.0]. 
- twig/twig v2.4.5 conflicts with knplabs/knp-menu[2.6.0]. 
- twig/twig v2.4.6 conflicts with knplabs/knp-menu[2.6.0]. 
- twig/twig v2.4.7 conflicts with knplabs/knp-menu[2.6.0]. 
- twig/twig v2.4.8 conflicts with knplabs/knp-menu[2.6.0]. 
- twig/twig v2.5.0 conflicts with knplabs/knp-menu[2.6.0]. 
- twig/twig v2.6.0 conflicts with knplabs/knp-menu[2.6.0]. 
- twig/twig v2.6.1 conflicts with knplabs/knp-menu[2.6.0]. 
- twig/twig v2.6.2 conflicts with knplabs/knp-menu[2.6.0]. 
- twig/twig v2.7.0 conflicts with knplabs/knp-menu[2.6.0]. 
- Installation request for knplabs/knp-menu (locked at 2.6.0) -> satisfiable by knplabs/knp-menu[2.6.0]. 
- Installation request for friendsofsymfony/ckeditor-bundle ^2.1 -> satisfiable by friendsofsymfony/ckeditor-bundle[2.1.0].
- Conclusion: don't install twig/twig v3.0.0|install twig/twig v2.0.0|install twig/twig v2.1.0|install twig/twig 2.2.0|install twig/twig v2.3.0|install twig/twig v2.3.1|install twig/twig v2.3.2|install twig/twig v2.4.0|install twig/twig v2.4.1|install twig/twig v2.4.2|install twig/twig v2.4.3|install twig/twig v2.4.4|install twig/twig v2.4.5|install twig/twig v2.4.6|install twig/twig v2.4.7|install twig/twig v2.4.8|install twig/twig v2.5.0|install twig/twig v2.6.0|install twig/twig v2.6.1|install twig/twig v2.6.2|install twig/twig v2.7.0|install twig/twig v2.7.1|install twig/twig v2.7.2|install twig/twig v2.7.3|install twig/twig v2.7.4|install twig/twig v2.8.0|install twig/twig v2.8.1 
- Conclusion: remove twig/twig v3.0.0|install twig/twig v2.0.0|install twig/twig v2.1.0|install twig/twig v2.2.0|install twig/twig v2.3.0|install twig/twig v2.3.1|install twig/twig v2.3.2|install twig/twig v2.4.0|install twig/twig v2.4.1|install twig/twig v2.4.2|install twig/twig v2.4.3|install twig/twig v2.4.4|install twig/twig v2.4.5|install twig/twig v2.4.6|install twig/twig v2.4.7|install twig/twig v2.4.8|install twig/twig v2.5.0|install twig/twig v2.6.0|install twig/twig v2.6.1|install twig/twig v2.6.2|install twig/twig v2.7.0|install twig/twig v2.7.1|install twig/twig v2.7.2|install twig/twig v2.7.3|install twig/twig v2.7.4|install twig/twig v2.8.0|install twig/twig v2.8.1 
- friendsofsymfony/ckeditor-bundle 2.1.0 requires twig/twig ^2.0 -> satisfiable by twig/twig[v2.0.0, v2.1.0, v2.10.0, v2.11.0, v2.11.1, v2.11.2, v2.11.3, v2.12.0, v2.12.1, v2.12.2, v2.2.0, v2.3.0, v2.3.1, v2.3.2, v2.4.0, v2.4.1, v2.4.2, v2.4.3, v2.4.4, v2.4.5, v2.4.6, v2.4.7, v2.4.8, v2.5.0, v2.6.0, v2.6.1, v2.6.2, v2.7.0, v2.7.1, v2.7.2, v2.7.3, v2.7.4, v2.8.0, v2.8.1, v2.9.0]
- Can only install one of: twig/twig[v2.10.0,v3.0.0]. 
- Can only install one of: twig/twig[v2.11.0,v3.0.0]. 
- Can only install one of: twig/twig[v2.11.1,v3.0.0]. 
- Can only install one of: twig/twig[v2.11.2,v3.0.0]. 
- Can only install one of: twig/twig[v2.11.3,v3.0.0]. 
- Can only install one of: twig/twig[v2.12.0,v3.0.0]. 
- Can only install one of: twig/twig[v2.12.1,v3.0.0]. 
- Can only install one of: twig/twig[v2.12.2,v3.0.0]. 
- Can only install one of: twig/twig[v2.9.0,v3.0.0].
- Installation request for twig/twig (locked at v3.0.0) -> satisfiable by twig/twig[v3.0.0].

Installation failed, reverting ./composer.json to its original content.
```

Here my composer.json
```
{
    "type": "project",
    "license": "proprietary",
    "require": {
        "php": "^7.2",
        "ext-ctype": "*",
        "ext-iconv": "*",
        "beberlei/doctrineextensions": "^1.2",
        "guzzlehttp/guzzle": "^6.4",
        "knplabs/knp-menu-bundle": "^2.3",
        "knplabs/knp-snappy-bundle": "^1.6",
        "mhujer/breadcrumbs-bundle": "^1.5",
        "phpoffice/phpspreadsheet": "^1.10",
        "predis/predis": "^1.1",
        "sensio/framework-extra-bundle": "^5.1",
        "snc/redis-bundle": "^3.2",
        "symfony/apache-pack": "^1.0",
        "symfony/asset": "4.4.*",
        "symfony/console": "4.4.*",
        "symfony/dotenv": "4.4.*",
        "symfony/expression-language": "4.4.*",
        "symfony/flex": "^1.3.1",
        "symfony/form": "4.4.*",
        "symfony/framework-bundle": "4.4.*",
        "symfony/http-client": "4.4.*",
        "symfony/intl": "4.4.*",
        "symfony/mailer": "4.4.*",
        "symfony/monolog-bundle": "^3.1",
        "symfony/orm-pack": "*",
        "symfony/process": "4.4.*",
        "symfony/security-bundle": "4.4.*",
        "symfony/serializer-pack": "*",
        "symfony/translation": "4.4.*",
        "symfony/twig-pack": "*",
        "symfony/validator": "4.4.*",
        "symfony/web-link": "4.4.*",
        "symfony/yaml": "4.4.*"
    },
    "require-dev": {
        "symfony/debug-pack": "*",
        "symfony/maker-bundle": "^1.0",
        "symfony/profiler-pack": "*",
        "symfony/test-pack": "*"
    },
    "config": {
        "preferred-install": {
            "*": "dist"
        },
        "sort-packages": true
    },
    "autoload": {
        "psr-4": {
            "App\\": "src/"
        }
    },
    "autoload-dev": {
        "psr-4": {
            "App\\Tests\\": "tests/"
        }
    },
    "replace": {
        "paragonie/random_compat": "2.*",
        "symfony/polyfill-ctype": "*",
        "symfony/polyfill-iconv": "*",
        "symfony/polyfill-php71": "*",
        "symfony/polyfill-php70": "*",
        "symfony/polyfill-php56": "*"
    },
    "scripts": {
        "auto-scripts": {
            "cache:clear": "symfony-cmd",
            "assets:install %PUBLIC_DIR%": "symfony-cmd"
        },
        "post-install-cmd": [
            "@auto-scripts"
        ],
        "post-update-cmd": [
            "@auto-scripts"
        ]
    },
    "conflict": {
        "symfony/symfony": "*"
    },
    "extra": {
        "symfony": {
            "allow-contrib": true,
            "require": "4.4.*"
        }
    }
}
```

Have you an idea?

Thanks

Tim

**Comment by mbabker** [comment]
> `Installation request for twig/twig (locked at v3.0.0)`

Your application has Twig 3.0 installed.  This bundle [isn't yet compatible](https://github.com/FriendsOfSymfony/FOSCKEditorBundle/pull/202) with it.  Downgrade Twig to 2.x and you should be OK.

**Comment by mattgxyz** [comment]
+1.  Downgrading twig leads to several other incompatibilities.  Would be nice if this was twig 4.x compatible.

**Comment by pam81** [comment]
I have same issue, it's not working with symfony 4.4

**Comment by jmsche** [comment]
I think this issue can be closed as #202 has been released for a while
